README.md

```diff
@@ -268,11 +270,12 @@ The ForageSDK exposes the following function to collect the EBT card number:
 This is an example of usage inside an ACC ViewModel:
 
 ```kotlin
-    fun onSubmit() = viewModelScope.launch {
+    fun onSubmit(panForageEditText: ForagePANEditText) = viewModelScope.launch {
         _isLoading.value = true
 
         val response = ForageSDK.tokenizeEBTCard(
             merchantAccount = merchantAccount,
+            panForageEditText = ForagePANEditText,
```

Comment:
```suggestion
    fun onSubmit() = viewModelScope.launch {
        _isLoading.value = true
        
        val tokenizeForagePanEditText = root?.findViewById<ForagePANEditText>(R.id.tokenizeForagePanEditText)

        val response = ForageSDK.tokenizeEBTCard(
            merchantAccount = merchantAccount,
            panForageEditText = tokenizeForagePanEditText,
```

Reply:
Maybe it can be done more than one way depending on the developers preference? I updated the README.md here with the pattern that I used in `sample-app/src/main/java/com/joinforage/android/example/ui/complete/flow/tokenize/FlowTokenizeFragment.kt`.

Let me know if there's something I'm missing that makes one way preferable!

Reply:
I think your approach is fine. It seems like you're passing the type on line 278, instead of the actual param.

```kotlin
            panForageEditText = ForagePANEditText,
````

Adding this line could make it immediately apparent that the "tokenizeForagePanEditText" is a view and not a regular object – given our [other snippets showcase XML](https://github.com/teamforage/forage-android-sdk#foragepanedittext). Your snippet LGTM too, so feel free to ignore this suggestion

```kotlin
        val tokenizeForagePanEditText = root?.findViewById<ForagePANEditText>(R.id.tokenizeForagePanEditText)
```

Reply:
this is a good idea addition! Will do

Reply:
Hey Danilo! This thread was not lost on me! I opened up a new PR to keep the README.md in sync with the latest changes, and [I believe I incorporated this feedback in the snippets](https://github.com/teamforage/forage-android-sdk/pull/105/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R261). Lmk any thoughts or feedback you on this and thanks again for surfacing this!